clang/include/clang/Driver/Options.td

```diff
@@ -6132,6 +6132,7 @@ def : Flag<["-"], "nogpulib">,
 def : Flag<["-"], "nocudalib">, Alias<no_offloadlib>;
 def gpulibc : Flag<["-"], "gpulibc">, Visibility<[ClangOption, CC1Option, FlangOption, FC1Option]>,
   HelpText<"Link the LLVM C Library for GPUs">;
+def nogpuflangrt : Flag<["-"], "nogpuflangrt">, Visibility<[ClangOption, CC1Option, FlangOption, FC1Option]>;
```

Comment:
could we use the -f form of the flag ?   and have both  
-fno-gpuflangrt   and   -fgpurflangrt  ?